source/adapters/hip/common.cpp

```diff
@@ -188,3 +178,21 @@ ur_result_t urGetLastResult(ur_platform_handle_t, const char **ppMessage) {
   *ppMessage = &ErrorMessage[0];
   return ErrorMessageCode;
 }
+
+ur_result_t GetHipFormatPixelSize(hipArray_Format Format, int *Size) {
```

Comment:
I know this is just refactoring existing code but does the `Size` argument need to be (pointer to) an integer. Most of the calculation using the size are with values of type `unsigned int` or `size_t` since size values are supposed to be greater or equal to 0. If it is a requirement to be an `int` due to something that I missed please let me know, I just thought that `unsigned int *Size` here would've been appropriate.

Reply:
Same applies to `imageElementByteSize`.

Reply:
Have updated these functions, cheers.